source/include/TypeAndDataManager.h

```diff
@@ -1011,7 +1010,6 @@ class DefaultPointerType : public PointerType
 {
  private:
     DefaultPointerType(TypeManagerRef typeManager, TypeRef type, void* pointer, PointerTypeEnum pointerType);
-    DefaultPointerType();
```

Comment:
Why not "DefaultPointerType() = delete;"?

Reply:
Because having a non default constructor declared already means that a default constructor won't be generated.